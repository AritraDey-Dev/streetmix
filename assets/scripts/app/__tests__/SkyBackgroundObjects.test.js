/* eslint-env jest */
import React from 'react'
import { SkyBackgroundObjects } from '../SkyBackgroundObjects'
import { mount } from 'enzyme'

describe('SkyBackgroundObjects', () => {
  it('renders', () => {
    const wrapper = mount(<SkyBackgroundObjects system={{ cssTransform: 'transform' }} environment="foo" />)
    expect(wrapper).toMatchSnapshot()
  })
})
