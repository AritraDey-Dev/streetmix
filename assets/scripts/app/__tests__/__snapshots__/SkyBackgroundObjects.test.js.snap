// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkyBackgroundObjects renders 1`] = `
<SkyBackgroundObjects
  environment="foo"
>
  <mockConstructor
    className="sky-background-objects"
  />
</SkyBackgroundObjects>
`;
